### DISCUSSION TITLE
폴더 엔티티 설계

### DISCUSSION THREAD

**Discussion by Gyaak** [Kernel360/F2-BAGUNI#107] [개발 토의]
# Folder 엔티티 설계
- 사용자 생성 시 자동으로 기본 폴더(최상위폴더 = 미분류 폴더)가 자동으로 생성된다.
- 폴더는 사용자에 속해야한다.
- 폴더는 다른 폴더에 속해야한다.

# Folder Entity Field
- Long id
- User owner
- Folder parent
- String name

# Discussion
- 추가로 필요한 필드가 있을지?
- 기본 폴더를 어떻게 정의할지

내일 당장 작업해야하는건이므로 빠른 확인 부탁해요 - 확인했으면 코멘트 or 이모지
@kimminkyeu @sangwonsheep

**Comment by kimminkyeu** [comment]
네 확인했습니다. 감사합니다.

**Comment by sangwonsheep** [comment]
확인 완료했습니다. 고생하셨어요 바로 작업 들어가도 좋을 것 같습니다.
